app-i18n/opencc/opencc-1.1.2.ebuild

```
@@ -1,8 +1,8 @@
 # Copyright 2010-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="7"
-PYTHON_COMPAT=( python3_{8,9} )
+EAPI="8"
+PYTHON_COMPAT=(python3_{8..10})
```

```suggestion
PYTHON_COMPAT=( python3_{8..10} )
```
do not remove spacing

---

These spaces are unnecessary.
They are also not included in canonical display of array variable using `declare -p`:
```
$ myarray=( a b c )
$ declare -p myarray
declare -a myarray=([0]="a" [1]="b" [2]="c")
```

There are different tastes, and there is nothing wrong with it.
Some people prefer:
```
if something; then
```
And other people prefer:
```
if something ; then
```
Some people prefer:
```
EAPI=8
```
And other people prefer:
```
EAPI="8"
```

---

Ok, I'll put it plain and simple: either you change it, or this won't go in. It's that simple.